views/index.jade

```diff
@@ -1,13 +1,14 @@
 extends layout
 
 block content
-
+    if user
     div.row
-
         // DISPLAY LOGIN OR REGISTER WHEN NOT LOGGED IN
-        if (!currentUser)
+        if (currentUser)
```

Comment:
If nothing is indented below this conditional, then there is no point in having it in the first place. Code must be indented below it for this conditional to have any useful functionality.

Reply:
this is an security issue #56 that needs to be patched